### ISSUE TITLE
Upgrading over existing app shows blank entry list

### ISSUE THREAD

**Issue by st3fan** [mozilla-lockbox/lockbox-ios#492]
I had the latest 1.0 from TestFlight  succesfully installed and configured on my phone. I am pretty certain that as of a few days ago that install worked properly and showed my logins.

I just upgraded to the first 1.1 build via TestFlight, and I am experiencing two oddities:

- The app opened without asking for FaceID. I put a piece of paper on top of the front camera to make sure to test this. https://github.com/mozilla-lockbox/lockbox-ios/issues/495
- The app opened with an empty logins list. Using pull to refresh does not refresh things. It did show the 'Sycing your Logins' screen.

I do have to admit that I am using iOS 12 Beta 2. But that has been pretty stable for me. I'll can try this upgrade path on a 11.4 device if that is useful.

**Comment by st3fan** [comment]
I tested this same path on my iOS 11.4 device. First I installed the last 1.0 build and logged in. I also set the Touch ID timer to 1 minute. That all works correctly: logins are synced and i have to get in via Touch ID after a minute.

Next I installed the 1.1 update on top. Touch ID still was requested so I am not sure if what I wrote in the first comment is iOS 12 or Face ID specific. But, the second issue is still there: my logins list is empty and doing a pull to refresh does not help.

**Comment by past** [comment]
I am seeing the empty list too. I don't see my FxA avatar and name, but I wasn't seeing it before 1.1 either. Disconnecting and reconnecting to FxA, didn't help. Tapping on the sort icon in the empty list doesn't do anything either.

**Comment by potch** [comment]
After upgrading to 1.1, My logins list was empty.

When viewing my account, it simply says "Firefox Account". I disconnected the app from my account and reconnected it. The app went gray with no "confirm your account" message until I remembered I had to confirm the login from my email. Then the app went through initial sync, resulting in the same empty login list and blank Account screen.

Fully deleting the app and re-installing from scratch fixes the problem.
